bridge-cli/src/fast_bridge_command.rs

```diff
@@ -0,0 +1,151 @@
+use crate::{combined_config, CliConfig, Network};
+use clap::Subcommand;
+use ethers_core::types::{Address, TxHash};
+use fast_bridge::{FastBridge, FastBridgeBuilder};
+use near_primitives::types::AccountId;
+use std::str::FromStr;
+
+#[derive(Subcommand, Debug)]
+pub enum FastBridgeSubCommand {
+    Transfer {
+        #[clap(short, long)]
+        token: String,
+        #[clap(short, long)]
+        amount: u128,
+        #[clap(short, long)]
+        fee: u128,
+        #[clap(short, long)]
+        eth_token_address: String,
+        #[clap(short, long)]
+        recipient: String,
+        #[clap(short, long)]
+        valid_till: u64,
+        #[command(flatten)]
+        config_cli: CliConfig,
+    },
+    TransferOnEth {
+        #[clap(short, long)]
+        token: String,
+        #[clap(short, long)]
+        amount: u128,
+        #[clap(short, long)]
+        recipient: String,
+        #[clap(short, long)]
+        nonce: u128,
```

Comment:
I think it would be more convenient to extract this information automatically from NEAR and only pass `nonce` and `unlock_recipient` here. At least having such an option.

Reply:
I agree. This can be done on top of the work I've done for the next PR so let me add it in the next one